packages/nimble-components/src/table-column/icon/tests/table-column-icon.mdx

```diff
@@ -16,15 +15,22 @@ import * as spinnerMappingStories from '../../../mapping/spinner/tests/mapping-s
 <Title of={tableColumnIconStories} />
 <Description of={tableColumnIconStories} />
 
-<Canvas of={tableColumnIconStories.iconColumn} />
-<ArgTypes of={tableColumnIconStories.iconColumn} />
+<DocsStory of={tableColumnIconStories.iconColumn} expanded={false} />
+<Controls of={tableColumnIconStories.iconColumn} />
 
-### Icon Mapping
+# Usage Docs
+
+## Blazor Usage
+
+When setting a child mapping elment's `Key` value to a string, you must wrap it in `@()` so that the compiler treats it as a string, e.g.
```

Comment:
```suggestion
When setting a child mapping element's `Key` value to a string, you must wrap it in `@()` so that the compiler treats it as a string, e.g.
```